basis/math/matrices/matrices.factor

```diff
@@ -197,11 +209,11 @@ ERROR: negative-power-matrix m n ;
 : cols ( seq matrix -- cols )
     '[ _ col ] map ; inline
 
-: set-index ( object pair matrix -- )
+: matrix-set-nth ( n pair matrix -- )
```

Comment:
I can see calling the element being put in the matrix ``n``, but maybe ``elt`` for consistency?  I dunno.

Reply:
Since technically, it doesn't have to be a number maybe.